packages/@react-spectrum/color/docs/ColorField.mdx

```diff
@@ -0,0 +1,150 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/color';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/color/package.json';
+
+```jsx import
+import {ColorField} from '@react-spectrum/color';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Color
+keywords: [color field, input]
+---
+
+# ColorField
+
+<p>{docs.exports.ColorField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorField']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<ColorField label="Primary Color" />
+```
+
+## Value
+
+A ColorField's `value` is empty by default, but an initial, uncontrolled, value can be provided using the `defaultValue` prop.
+Alternatively, a controlled value can be provided using the `value` prop. These values must begin with a `#`.
```

Comment:
Do we wanna mention that it should be a valid hex color code? (maybe a bit too obvious tho)

Reply:
Good question, I wasn't sure if mentioning that they have to start with a # is specific enough or not... saying a valid hex color code would probably be more specific and better.